src/components/Menu.js

```diff
@@ -105,6 +105,12 @@ type MenuProps = {
   width: number | null;
   /** Any CSS style you need to apply to the menu. Optional. */
   style: any;
+  /**
+   * Property that contains the value for data-test attribute
+   * added to elements to be uniquely identified by testing tools
+   * like Selenium.
+   */
+  dataTestValue?: string | null;
```

Comment:
I'm wondering if it makes sense to call this `dataTest` instead of `dataTestValue` since the eventual HTML attribute is called `data-test`? This would be in keeping with other pass-through things like `id` or `style` where the React component property has the same name as the HTML attribute...